javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/DeclarativeConfigPropertiesBridge.java

```diff
@@ -45,54 +46,51 @@
  *         string_key: value
  * </pre>
  */
-final class StructuredConfigPropertiesBridge implements ConfigProperties {
+final class DeclarativeConfigPropertiesBridge implements ConfigProperties {
 
   private static final String OTEL_INSTRUMENTATION_PREFIX = "otel.instrumentation.";
 
   // The node at .instrumentation.java
-  private final StructuredConfigProperties instrumentationJavaNode;
+  private final DeclarativeConfigProperties instrumentationJavaNode;
 
-  StructuredConfigPropertiesBridge(StructuredConfigProperties rootStructuredConfigProperties) {
-    instrumentationJavaNode =
-        rootStructuredConfigProperties
-            .getStructured("instrumentation", empty())
```

Comment:
@jack-berg I had to remove this line to get the modified DeclarativeConfigPropertiesBridgeTest from https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13463/files passing. Is this correct?

Reply:
It looks correct but should rename `rootStructuredConfigProperties` to `instrumentationNode` (or equivalent). 

AgentListener now obtains the `DeclarativeConfigProperties` representing the `.instrumentation` node via `ConfigProvider#getInstrumentationConfig()`.